build-system/global-configs/experiments-config.json

```diff
@@ -15,5 +15,12 @@
     "expirationDateUTC": "2019-10-10",
     "defineExperimentConstant": "_RTVEXP_INABOX_LITE"
   },
-  "experimentC": {}
+  "experimentC": {
+    "name": "CEv1",
+    "environment": "AMP",
+    "command": "gulp dist --defineExperimentConstant=CUSTOM_ELEMENTS_V1",
+    "issue": "https://github.com/ampproject/amphtml/issues/17243",
+    "expirationDateUTC": "2020-12-31",
```

Comment:
I hope we don't take another year to ship. How about +3 months?

Reply:
Lol. I was more annoyed by the recent build breaks. I don't know why we have this defined here.